### PR TITLE
Update 400-api-reference.mdx

### PR DESCRIPTION
**Pull request by nikolasburk**

### CODE REVIEW

File: content/300-accelerate/400-api-reference.mdx [hunk 1]
```diff
@@ -113,3 +113,9 @@ interface AccelerateInfo {
...
 | `signature`    | `String`                                                     | The unique signature of the Prisma operation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+
+## Errors
+
+Accelerate-related errors start with `P6xxx`.
```
Comment (nikolasburk) [inline content/300-accelerate/400-api-reference.mdx]: this line is quoted from the [actual error reference](https://www.prisma.io/docs/orm/reference/error-reference#accelerate):

image

Comment (ankur-arch) [reply content/300-accelerate/400-api-reference.mdx]: LGTM 👍 , maybe both places should read `Prisma Accelerate` rather than Accelerate.

Comment (ankur-arch) [inline content/300-accelerate/400-api-reference.mdx]:
```suggestion
Prisma Accelerate-related errors start with `P6xxx`.
```

Comment (nikolasburk) [reply content/300-accelerate/400-api-reference.mdx]: I don't like how this looks but ChatGPT says it's the way to go 😂 

image

Comment (nikolasburk) [reply content/300-accelerate/400-api-reference.mdx]: changed "Accelerate" to "Prisma Accelerate" on the other page as well: https://github.com/prisma/docs/pull/5577

**Pull request merged**